Handling Misscrambles (#56)
CarterKoala/wca-regulations-wrc: wca-regulations.md

- 11f) Decisions about an incident may be supported with video or photographic analysis, at the discretion of the WCA Delegate.
- 11g) The WCA Delegate must ensure that copies of the Regulations and Guidelines are available (e.g. printed, digital, or accessible via internet) to officials and competitors for consultation on any incidents.
- 11h) A WCA Delegate may require competitors in serious violation of [Regulation 2k3](regulations:regulation:2k3) to leave the competition venue, taking into account the seriousness of the situation and the best possible course of action. If the competitor refuses to do so, they may face disciplinary action in the WCA.
- 11i) If a competitor finds that their puzzle was not scrambled correctly after completing their solve in an event not listed as an exception in [Regulation 4g1](regulations:regulation:4g1), the following regulations apply:
    - 11i1) For any result that is a regional record or personal record in the top 50 results of the world ranking, an extra attempt must be granted before the end of the competition to replace the affected attempt. If no extra attempt is granted, the attempt is disqualified (DNF).

>     - 11i2) For any result achieved by solving the puzzle in less moves than the limits in [Regulation 4b3](regulations:regulation:4b3), an extra attempt must be granted before the end of the competition to replace the affected attempt. If no extra attempt is granted, the attempt is disqualified (DNF).

Reviewer: ```suggestion
    - 11i2) For any result achieved by solving the puzzle in fewer moves than the limits in [Regulation 4b3](regulations:regulation:4b3), an extra attempt must be granted before the end of the competition to replace the affected attempt. If no extra attempt is granted, the attempt is disqualified (DNF).
```

    - 11i3) For any result achieved on a puzzle deemed to be intentionally scrambled incorrectly, an extra attempt should be granted, at the discretion of the WCA Delegate.

>     - 11i4) For all other cases, the result stands.

Author: Is "stands" the right wording here?

Reviewer: Fine IMO. We could potentially express this as no penalty instead if preferred.

>     - 11i4) For all other cases, the result stands.

Reviewer 2: Same idea with a few changes and major championship podiums:
```
- 11i) For events not listed as an exception in [Regulation 4g1](regulations:regulation:4g1), if it is found that a competitor's puzzle was not scrambled correctly after they have completed their solve, the following regulations apply:
    - 11i1) In the following cases, an extra attempt must be granted before the end of the competition to replace the affected attempt. If no extra attempt is granted, the attempt is disqualified (DNS):
        - 11i1a) The result is a regional record or a personal record in the top 50 results of the world ranking.
        - 11i1b) The result is achieved in a final round at a major championship and it is necessary for the competitor to get into the podium.
        - 11i1c) The result was achieved by solving the puzzle in fewer moves than the limits defined in [Regulation 4b3](regulations:regulation:4b3).
    - 11i2) For all other cases, an extra attempt should be granted during the competition. If no extra attempt is granted, the result may [¿or should?] stay [¿, at the discretion of the WCA Delegate or the WRC?].
```

11i: I think "if it is found that (...)" is more general than "if a competitor finds that their (...)".
11i1: Changed to avoid repeating the same in three regulations.
11i1b: I guess "major championship" may be replaced with something like "national, continental or world championship". Idk, just an idea to cover one of the points Carter mentioned in the description of the PR.

Author: ```suggestion
- 11i) If it is found that a competitor's puzzle was not scrambled correctly after they have completed their solve the following regulations apply:
    - 11i1) In the following cases, except for an extra attempt must be granted before the end of the competition to replace the affected attempt. If no extra attempt is granted, the result will be considered DNS:
        - 11i1a) The result is a regional record or a personal record in the top 50 results of the world ranking.
        - 11i1b) The result is achieved in a final round of a Championship.
        - 11i1c) The result was achieved by solving the puzzle in fewer moves than the limits defined in [Regulation 4b3](regulations:regulation:4b3).
        - 11i1d) Exception: For 5x5x5 Cube, 6x6x6 Cube, 7x7x7 Cube, Megaminx, and 3x3x3 Multi-Blind, an extra attempt is not necessary to be granted under the circumstances described above.
    - 11i2) For any result achieved on a puzzle deemed to be intentionally scrambled incorrectly, an extra attempt should be granted, at the discretion of the WCA Delegate.
    - 11i3) For all other cases, the result stands.
```

I like your ideas - but a couple notes:

11i1: Minor wording change (to match with other DNS results in the regs)

11i1b: I think we can just call it a "Championship" since that is the wording the WCRP uses, and maybe a guideline to clarify this only counts for National, Continental, or World championships that are marked as championships. Also, I think it makes sense to remove the last clause "and it is necessary for the competitor to get into the podium" since a misscramble can make you miss out on a podium. 

11i2/11i3: Not sure what the current idea is on how to treat the non-notable misscrambles

This still doesn't answer some questions though:
- What do we do with duplicate scrambles
- Clock scramblers forgetting to do a y2 is a pretty unique misscramble that should probably not be allowed

Reviewer 2: And maybe something like:
```
- 11i2+) [CLARIFICATION] The result should not be allowed to stay if the incorrect scramble is significantly easier than the original scramble (e.g. ...).
```

Author: ```suggestion
- 11i) If it is found that a competitor's puzzle was not scrambled correctly after they have completed their solve, the following regulations apply:
    - 11i1) In the following cases, an extra attempt must be granted to replace the affected attempt. If no extra attempt is granted, the result will be considered DNS:
        - 11i1a) The result is a regional record or a personal record in the top 50 results of the world ranking.
        - 11i1b) The result is achieved in a final round of a Championship.
        - 11i1c) The result was achieved by solving the puzzle in fewer moves than the limits defined in [Regulation 4b3](regulations:regulation:4b3).
        - 11i1d) Exception: For 5x5x5 Cube, 6x6x6 Cube, 7x7x7 Cube, Megaminx, and 3x3x3 Multi-Blind, an extra attempt is not necessary to be granted under the circumstances described above.
    - 11i2) For all other cases, if the incorrect scramble was not significantly easier than the original scramble, the result may stand. Otherwise, an extra attempt must be granted, at the discretion of the WCA Delegate.
```

Second suggestion based on what was mentioned in the Slack thread. Definitely needs some guidelines too, but I am liking this a bit more.

Reviewer 2: > 11i2/11i3: Not sure what the current idea is on how to treat the non-notable misscrambles

I'm not entirely sure why we want the proposed wording of 11i2. That's why I changed 11i2/11i3 to this in my comment above:
```
- 11i2) For all other cases, an extra attempt should be granted during the competition. If no extra attempt is granted, the result may stand [¿, at the discretion of the WCA Delegate or the WRC?].
```

11i1d: Do we want to repeat these exceptions instead of mentioning 4g1 + MBLD?

> What do we do with duplicate scrambles

Tbh, I'm for not allowing duplicates at all. I know this might be kinda harsh, but the potential to exploit the situation is always there for any competitor getting a duplicate, and we would need to trust they word if they say "I didn't notice, so I did a different solve".

> Clock scramblers forgetting to do a y2 is a pretty unique misscramble that should probably not be allowed

I think this would be covered by the 11i2 approach I mentioned here.

Reviewer 2: Yeah, it's looking good :)

What about reducing this
```
- 11i2) For all other cases, the following regulations apply:
    - 11i2a) If the incorrect scramble was not significantly easier than the original scramble, the result may stand, at the discretion of the WCA Delegate.
    - 11i2b) If the incorrect scramble was significantly easier than the original scramble, an extra attempt must be granted, at the Discretion of the WCA Delegate.
```
to this?
```
- 11i2) For all other cases, if the incorrect scramble was not significantly easier than the original scramble, the result may stand. Otherwise, an extra attempt must be granted, at the discretion of the WCA Delegate.
```
Either way looks good to me, I'm just considering an option with less numbered regulations.

- 11f) Decisions about an incident may be supported with video or photographic analysis, at the discretion of the WCA Delegate.
- 11g) The WCA Delegate must ensure that copies of the Regulations and Guidelines are available (e.g. printed, digital, or accessible via internet) to officials and competitors for consultation on any incidents.
- 11h) A WCA Delegate may require competitors in serious violation of [Regulation 2k3](regulations:regulation:2k3) to leave the competition venue, taking into account the seriousness of the situation and the best possible course of action. If the competitor refuses to do so, they may face disciplinary action in the WCA.
- 11i) If it is found that a competitor's puzzle was not scrambled correctly after they have completed their solve, the following regulations apply:
    - 11i1) In the following cases, an extra attempt must be granted to replace the affected attempt. If no extra attempt is granted, the result will be considered DNS:

>         - 11i1a) The result is a regional record or a personal record in the top 50 results of the world ranking.

Reviewer: ```suggestion
        - 11i1a) The individual result, or the average or mean the result is part of, is a regional record or a personal record in the top 50 results of the world ranking.
```
To capture the impact of non-notable singles on notable averages

Reviewer 2: What if the affected attempt is the worst result in a notable average?

Author: > What if the affected attempt is the worst result in a notable average?

Then it should still be replaced with an extra. This would never hurt the competitor, and makes the average "more legitimate" as all the scrambles would be correct. I like Hex's suggestion.

>         - 11i1b) The result is achieved in a final round of a Championship.

Reviewer: ```suggestion
        - 11i1b) The result is achieved in a final round of a National Championship, Continental Championship or World Championship.
```
Align with progression PR by listing championship types rather than having to separate define what 'Championship' means in general.

        - 11i1c) The result was achieved by solving the puzzle in fewer moves than the limits defined in [Regulation 4b3](regulations:regulation:4b3).

>         - 11i1d) Exception: For 5x5x5 Cube, 6x6x6 Cube, 7x7x7 Cube, Megaminx, and 3x3x3 Multi-Blind, an extra attempt is not necessary to be granted under the circumstances described above.

Reviewer 2: I still think it would make it cleaner to mention 4g1 in 11i1 (instead of adding 11i1d). I don't remember if there was any discussion to move it to an independent reg.

Author: I couldn't write it in a cleaner than this, since MBLD is included. Certainly open to suggestions though

>     - 11i2) For all other cases, if the incorrect scramble was not significantly easier than the original scramble, the result may stand. Otherwise, an extra attempt must be granted, at the discretion of the WCA Delegate.

Reviewer: ```suggestion
    - 11i2) For all other cases, the result may stand. Exception: the WCA Delegate may grant an extra attempt at their discretion if they believes this provides a significantly fairer outcome (e.g. the incorrect scramble received was extremely easy and the competitor achieved a new personal record, the incorrect scramble impacted the podium for an event, or the puzzle was deliberately scrambled incorrectly).
```
Trying to narrow the cases in which "regular" misscrambles get replaced and do away with the challenging "significantly easier" analysis.

Reviewer 2: I think it looks good. Note the "if they believes" (I think it should be "believe", right?).

>     - 11i2) For all other cases, the result may stand. Exception: the WCA Delegate may grant an extra attempt at their discretion if they believes this provides a significantly fairer outcome (e.g. the incorrect scramble received was extremely easy and the competitor achieved a new personal record, the incorrect scramble impacted the podium for an event, or the puzzle was deliberately scrambled incorrectly).

Author: ```suggestion
    - 11i2) For all other cases, the result may stand. Exception: the WCA Delegate may grant an extra attempt at their discretion if they believe this provides a significantly fairer outcome (e.g. the incorrect scramble received was extremely easy and the competitor achieved a new personal record, the incorrect scramble impacted the podium for an event, or the puzzle was deliberately scrambled incorrectly).
```

- 11j) If a competitor incorrectly receives a scramble that they have already received (i.e. a duplicate scramble), the following regulations apply:

>     - 11j1) If the competitor believes they received a duplicate scramble in inspection, they should stop their attempt and call over a WCA Delegate without applying any moves to the puzzle. A competitor should not be penalized for incorrectly reporting a duplicate scramble.

Reviewer 3: Do we want to clarify that the delegates can penalize the competitor if the competitor is trying to get an advantage?

Reviewer 2: Yeah. IMO "a competitor should not be penalized for incorrectly reporting a duplicate scramble" only if the confusion is understandable for any reason.

Author: ```suggestion
    - 11j1) If the competitor believes they received a duplicate scramble in inspection, they should stop their attempt and call over a WCA Delegate without applying any moves to the puzzle. A competitor should not be penalized for incorrectly reporting a duplicate scramble if the WCA Delegate believes their report is genuine.
```
Maybe this?

Reviewer 2: What about the classic "at the discretion of the WCA Delegate"?

Author: ```suggestion
    - 11j1) If the competitor believes they received a duplicate scramble in inspection, they should stop their attempt and call over a WCA Delegate without applying any moves to the puzzle. A competitor should not be penalized for incorrectly reporting a duplicate scramble genuinely, at the discretion of the WCA Delegate.
```
I like this

>     - 11j2) If the error is noticed during the group, the WCA Delegate should make an attempt to give the competitor the correct scramble. An extra attempt may be given if the WCA Delegate is not certain what the correct scramble should be.

Reviewer 2: ```suggestion
    - 11j2) If the error is noticed during the group, the WCA Delegate should try to give the competitor the correct scramble. An extra attempt may be given if the WCA Delegate is not certain what the correct scramble should be.
```
Avoiding "make an attempt" so it's not confused with a solving attempt.

Author: ```suggestion
    - 11j2) If the error is noticed during the group, the WCA Delegate should make an effort to give the competitor the correct scramble. An extra attempt may be given if the WCA Delegate is not certain what the correct scramble should be.
```
"Try" seems too informal, how about this?
